app/stores/AppStore.js

```diff
@@ -75,7 +75,7 @@ export default class AppStore extends Store {
   @computed get currentTheme(): string {
     const { result } = this.getThemeRequest.execute();
     if (this.isCurrentThemeSet) return result;
-    return 'themeDefault'; // default
+    return environment.isMainnet() ? 'theme1' : 'themeDefault'; // default
```

Comment:
How about changing the theme file names to their names like `/themes/daedalus/cardano.js` and `/themes/daedalus/lightBlue.js`?

Reply:
@DominikGuzei there is a task for that: https://issues.serokell.io/issue/DAEF-451
@darko-mijic and already discussed this and it has been decided this optimization will be carried out after ALL currently open PRs are merged into master (in order to avoid merge conflicts)...